t/ui/18-tests-details.t

```diff
@@ -641,6 +641,13 @@ subtest 'additional investigation notes provided on new failed' => sub {
         'investigation status content shown as table');
 };
 
+subtest 'alertbox shown if first_bad test available' => sub {
+    $driver->get('/tests/99940');
+    wait_for_ajax(msg => 'details tab for job 99940 loaded to test investigation');
+    $driver->find_element_by_link_text('Investigation')->click;
+    $driver->find_element("//div[\@class='alert alert-info']", 'xpath')->text_like(qr/Investigate the first bad test directly: 99938/);
```

Comment:
I suggest to combine this test with the above subtest, e.g. like:

```
subtest 'additional investigation notes provided on new failed' => sub {
    $driver->get('/tests/99940');
    wait_for_ajax(msg => 'details tab for job 99940 loaded to test investigation');
    $driver->find_element_by_link_text('Investigation')->click;
    my $box = $driver->find_element("//div[\@class='alert']", 'xpath'), 'alertbox');
    ok $box, 'hint for "first bad" shown';
    $box->(… get link from box …)->click;
    # TODO ensure that the link brought us to the investigation tab directly
    ok($driver-&gt;find_element('table#investigation_status_entry')-&gt;text_like(qr/No result dir/);
    # TODO ensure that no "hint to 'first bad'" shows up when already on first bad
```

Reply:
That was what I tried first. But the above subtest for the test 99947 has this line:
```
$driver->find_element('#clones a')->click;
```
But 99940 doesn't have such a link.
Do you suggest to remove the test for `#clones a`?

Reply:
I checked and I don't remember why I added that line in https://github.com/perlpunk/openQA/commit/ef6223b252da205f5fca0eb95593e3e59ea4bd5f#diff-752f62785dc57e09d41a9dd89bdc34e67f66a81de9248dda0482be71e8ab4d3aR535 . Go ahead and remove it.

Reply:
I just can't get it working.
The first_bad link for 99940 links to 99938.
But for this test, "No result dir" doesn't show up in the investigation tab.

Reply:
I pushed some changes.
For now I kept the two subtests.
Maybe you can try it out on your side to get it working with just one.